README.md

```diff
@@ -280,6 +280,7 @@ wsa://com.apple.android.music
 | Plex Dash | 1.1.1 | 11 | ❌ | App crashes after splash screen
 | Plexamp | 3.8.2 | 11 | ⚠️ | Layout and app orientation issues
 | Pocket | 7.56.0.0 | 11 | ⚠️ | Unable to log in with a Firefox account, instant (push) syncing is unavailable
+| Polar Flow | 6.12.0 | 12 | ✅ | Not able to sync via Bluetooth since WSA doesn't support it. | Syncing via web and interaction is otherwise fine.
```

Comment:
```suggestion
| Polar Flow | 6.12.0 | 12 | ✅ | Not able to sync via Bluetooth since WSA doesn't support it. | Syncing via web and interaction is otherwise fine.
```